extension/spark-doris-connector/pom.xml

```diff
@@ -168,6 +211,10 @@
                             <pattern>com.google.flatbuffers</pattern>
                             <shadedPattern>org.apache.doris.flatbuffers</shadedPattern>
```

Comment:
Maybe we should unify the `shaded` package name,  such as:

xxx -> org.apache.doris.shaded.xxxxx